### ISSUE TITLE
There is a bug when reinstalling Langflow1.0.0a37 through a local folder when Langflow1.0.5 is already available.

### ISSUE THREAD

**Issue by zzleaning** [langflow-ai/langflow#2472]
How to reinstall version 1.0.0a37 of Langflow1.0.0a37 when there is already version 1.0.5 of Langflow1.0.5?
I encountered a bug after trying the following
When there is already version 1.0.5  of Langflow1.0.5, reinstall Langflow1.0.0a37 through a local folder.
![1 0 37](https://github.com/langflow-ai/langflow/assets/111033943/06922cef-cfd8-4d48-8361-db7ca7c9b80e)

After installation, there is an error message stating that the frontend file is missing. 
![1111](https://github.com/langflow-ai/langflow/assets/111033943/4c9aece1-e748-480e-b195-2bf3102c0397)

Move the frontend of the local file to the running path directory and try running it again
An error occurred while running.
![22222222222](https://github.com/langflow-ai/langflow/assets/111033943/7a5ed822-7412-44b9-814e-900e13c436c5)

**Comment by zzleaning** [comment]
After uninstalling the current version, reinstall the local Langflow 1.0.37 version, allowing for no folder loss error but still encountering errors.
![77777777](https://github.com/langflow-ai/langflow/assets/111033943/4766aa46-8d70-47c9-a590-d0f65bd51fea)

**Comment by zzleaning** [comment]
@dosu After clearing the cache and deleting files, run migration to resolve the issue with the following error message.
![52442](https://github.com/langflow-ai/langflow/assets/111033943/d119453e-2256-45bf-851d-e0dd678a278b)

Afterwards, attempting to run langflow directly resulted in the following error message.
![123](https://github.com/langflow-ai/langflow/assets/111033943/e98f1477-5131-418b-a0a5-89a8b8b78dfe)

**Comment by zzleaning** [comment]
@dosu I hope to use the old version Langflow 1.0.0a37. How to install it directly.

**Comment by zzleaning** [comment]
Install Langflow 1.0.0a37 directly using instructions
![213](https://github.com/langflow-ai/langflow/assets/111033943/f40b2833-902b-429b-991e-84bcfa2c7246)

The same error message still appears after running
![65546](https://github.com/langflow-ai/langflow/assets/111033943/bd369b3e-3fcf-4834-bd53-544b9cb4352f)

**Comment by nicoloboschi** [comment]
@zzleaning is there a specific reason why you want to use a deprecated and unmaintained version of langflow?

in any case, I suggest you to remove the entire langflow directory in the conda environment and try with a fresh install. 
Also setting LANGFLOW_LOG_LEVEL=debug in the env variable might help understanding the actual problem

**Comment by zzleaning** [comment]
> @zzleaning is there a specific reason why you want to use a deprecated and unmaintained version of langflow?
> 
> in any case, I suggest you to remove the entire langflow directory in the conda environment and try with a fresh install. Also setting LANGFLOW_LOG_LEVEL=debug in the env variable might help understanding the actual problem

Thanks for your answer. I am using this version only because the previously installed version was missing many components. Perhaps there was an issue during the first configuration, but I overlooked it. At present, I am still a beginner in learning how to use Langflow, and I have not fully understood many things, so I am trying to switch to the version that my leader is currently using. Yesterday at the end, I created a new conda environment and reinstalled Langflow, and now it can be used normally. Thank you again!
